.github/workflows/connector-tests.yml

```diff
@@ -119,7 +121,7 @@ jobs:
         if: steps.no_changes.outputs.status != 'cancelled'
         with:
           repository: airbytehq/airbyte
-          ref: master
+          ref: aj/airbyte-ci/allow-report-uploads-with-fewer-permissions
```

Comment:
TODO: Revert to `master` before merging, after this has merged:

- https://github.com/airbytehq/airbyte/pull/48824